GitTrends/ViewModels/RepositoryViewModel.cs

```diff
@@ -118,7 +129,14 @@ public bool IsRefreshing
             get => _isRefreshing;
             set => SetProperty(ref _isRefreshing, value);
         }
+        string _totalButtonText = "TOTAL";
 
+        
+        public string TotalButtonText
+        {
+            get { return _totalButtonText; }
```

Comment:
Let's use expression bodied members to match the other properties in the code base

```csharp
get => _totalButtonText; 
```